source/dofs/dof_renumbering.cc

```diff
@@ -689,7 +690,12 @@ namespace DoFRenumbering
                                             dof_handler.end(),
                                             component_order_arg,
                                             false);
-    if (result == 0)
+    (void)result;
```

Comment:
Since you don't use the `result` object, just discard the return value of the function call above.